### DISCUSSION TITLE
tracking/lines

### DISCUSSION THREAD

**Discussion by RAFALCO** [opengisch/QField#5884] [Q&A]
Sometimes after stopping the "tracking" I want to start a new one. In the majority of cases qfield behaves as if I had asked to "resume the route" when I want to individualize the new one. Sometimes it works. 
i use gpkg file.
on all versions of qfield and even with the "bees" example projecton.
This happens even if I turn Qfield off and on again.
Where does the problem come from?
![tracking erreur2](https://github.com/user-attachments/assets/f12e8d7c-bca7-4823-88f9-c2961abb34ff)

**Comment by nirvn** [comment]
@RAFALCO , could you provide us with a screencast and steps to reproduce? I've so far not been able to recreate this issue locally.

**Comment by RAFALCO** [comment]
The problem is resolved with version 3.5 of QField.
